hive-metastore/src/main/java/org/apache/iceberg/hive/HiveViewOperations.java

```diff
@@ -299,7 +299,10 @@ private void setHmsTableParameters(
           BaseMetastoreTableOperations.PREVIOUS_METADATA_LOCATION_PROP, currentMetadataLocation());
     }
 
-    setSchema(metadata.schema(), parameters);
+    HMSTablePropertyHelper.setSchema(
+        metadata.schema(),
+        parameters,
+        conf.getLong(HIVE_TABLE_PROPERTY_MAX_SIZE, HIVE_TABLE_PROPERTY_MAX_SIZE_DEFAULT));
```

Comment:
Are there any uses left or `maxHiveTablePropertySize`?
Why are we parsing it here instead of using the method?

Reply:
Good catch, fixing it!